openapi/geonames/client.bal

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the year to 2022. Check all possible places

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/8afbfba4dcd9e01f94c50b884aed73cdf5b490b6